integration/mediation-tests/tests-service/src/test/java/org/wso2/carbon/esb/rest/test/header/ESBJAVA2283ReturnContentTypeTestCase.java

```diff
@@ -1,99 +1,109 @@
 import org.testng.annotations.Test;
 import org.wso2.esb.integration.common.utils.ESBIntegrationTest;
 import org.wso2.esb.integration.common.utils.ESBTestCaseUtils;
 
 public class ESBJAVA2283ReturnContentTypeTestCase extends ESBIntegrationTest {
+
     private Log log = LogFactory.getLog(ESBJAVA2283ReturnContentTypeTestCase.class);
-	private HttpServer server = null;
 
     @BeforeClass(alwaysRun = true)
     public void init() throws Exception {
-	    super.init();
-	    String relativePath = "/artifacts/ESB/synapseconfig/esbjava2283/api.xml";
-	    ESBTestCaseUtils util = new ESBTestCaseUtils();
-	    relativePath = relativePath.replaceAll("[\\\\/]", "/");
-	    OMElement apiConfig = util.loadResource(relativePath);
-	    addApi(apiConfig);
+
+        super.init();
+        String relativePath = "/artifacts/ESB/synapseconfig/esbjava2283/api.xml";
+        ESBTestCaseUtils util = new ESBTestCaseUtils();
+        relativePath = relativePath.replaceAll("[\\\\/]", "/");
+        OMElement apiConfig = util.loadResource(relativePath);
+        addApi(apiConfig);
     }
 
     @Test(groups = {"wso2.esb"}, description = "test return content type")
     public void testReturnContentType() throws Exception {
-	    int port = 8089;
-	    String contentType = "text/xml";
-	    server = HttpServer.create(new InetSocketAddress(port), 0);
-	    server.createContext("/gettest", new MyHandler());
-	    server.setExecutor(null); // creates a default executor
-	    server.start();
-	    DefaultHttpClient httpclient = new DefaultHttpClient();
-	    String url = "http://localhost:8480/serviceTest/test";
-	    HttpGet httpGet = new HttpGet(url);
-	    HttpResponse response = null;
-	    try {
-		    response = httpclient.execute(httpGet);
-	    } catch (IOException e) {
-		    log.error("Error Occurred while sending http get request. " + e);
-	    }
-	    log.info(response.getEntity().getContentType());
-	    log.info(response.getStatusLine().getStatusCode());
-
-	    assertEquals(response.getFirstHeader("Content-Type").getValue(), contentType,
-	                 "Expected content type doesn't match");
-	    assertEquals(response.getStatusLine().getStatusCode(), 200, "response code doesn't match");
-
-	    server.stop(0);
+
+        int port = 8089;
+        HttpServer server = null;
+        String contentType = "text/xml";
+        server = HttpServer.create(new InetSocketAddress(port), 0);
+        server.createContext("/gettest", new MyHandler());
+        server.setExecutor(null); // creates a default executor
+        server.start();
+        DefaultHttpClient httpclient = new DefaultHttpClient();
+        String url = "http://localhost:8480/serviceTest/test";
+        HttpGet httpGet = new HttpGet(url);
+        HttpResponse response = null;
+        try {
+            response = httpclient.execute(httpGet);
+        } catch (IOException e) {
+            log.error("Error Occurred while sending http get request. " + e);
```

Comment:
Shall we replace plus sign with comma since, we need to send exception as argument to the log.error function.